xlators/cluster/afr/src/afr-inode-write.c

```diff
@@ -2531,6 +2541,12 @@ afr_fsync(call_frame_t *frame, xlator_t *this, fd_t *fd, int32_t datasync,
     if (!local->xdata_req)
         goto out;
 
+    ret = dict_set_int32_sizen(local->xdata_req, GLUSTERFS_INODELK_COUNT, 0);
```

Comment:
Any reason to go with this instead of GLUSTERFS_INODELK_DOM_COUNT with domain this->name like in inode-write?

Reply:
I did it to release the locks irrespective of the domain (shd, data, metadata) the blocked locks are in, but thinking more, since `afr_fsync` only takes data domain lock,  we can limit the detection to that domain alone with `this->name`